VMTests/vmConstantinopleTests/extcodehash3.json

```diff
@@ -25,31 +24,34 @@
         "post": {
             "0x0f572e5295c57f15886f9b263e2f6d2d6c7b5ec6": {
                 "balance": "0x152d02c7e14af6800000",
-                "code": "0x730000000000000000000000000000000000000001600055",
+                "code": "0x7300000000000000000000000000000000000000013f600055",
                 "nonce": "0x00",
                 "storage": {
                     "0x00": "c5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470"
                 }
             },
-            "0x0f572e5295c57f15886f9b263e2f6d2d6c7b5001": {
+            "0x0000000000000000000000000000000000000001": {
```

Comment:
All passed in SputnikVM, though I'm concerned about the address collision for `0x0..1` with the precompiled contract `H160`.

Reply:
Noted, changed `0x0...1` -> `0x0..deadbeef` to avoid that possibility.